build.gradle.kts

```diff
@@ -1,49 +1,52 @@
-import org.jetbrains.changelog.markdownToHTML
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 fun properties(key: String) = project.findProperty(key).toString()
 
 plugins {
+    val kotlinVersion = "1.5.10"
+    // Spring
+    id("io.spring.dependency-management") version "1.0.9.RELEASE"
+    id("org.springframework.boot") version "2.3.0.RELEASE" apply false
```

Comment:
Nieuwere versie beschikbaar: 2.5.4

Reply:
Hier breekt hij op.

Reply:
Dat zou kunnen, het is namelijk twee minor version verder, waar mogelijk breaking changes in zitten. Mooiste is wel als het werkt met de op dit moment meest recente versie van Spring boot. Anders loop je op voorhand al achter de feiten aan.